hieradata/common.yaml

```diff
@@ -1 +1,19 @@
-classes: []
+classes:
+    - ocf::apt
+    - ocf::auth
+    - ocf::autologout
+    - ocf::groups
+    - ocf::kerberos
+    - ocf::ldap
+    - ocf::locale
+    - ocf::motd
+    - ocf::munin::node
+    - ocf::networking
+    - ocf::packages
+    - ocf::puppet
+    - ocf::rootpw
+    - ocf::staff_users
+    - ocf::systemd
+    - ocf::utils
+
+packages: {}
```

Comment:
I'm not sure what the best way would be, but I think making a separate manifest for it, or a conditional somewhere would be better than doing this. Maybe I'm exaggerating how much this would be abused, but I do see it as a source of confusion because it could be so broadly used.